src/client/containers/QuizPage/QuizPage.component.js

```diff
@@ -21,23 +77,55 @@ export const QuizPage = () => {
           </div>
           <div className="question_page">
             <QuestionComponent
-              imageSrc={image1}
-              question="When visiting a website, what is that you are most interested in?"
+              imageSrc={QuizQuestions[currentOn].image}
+              question={QuizQuestions[currentOn].question}
             />
-            <div className="quizans_page">
+            <div>
               <QuizAnswers
-                isAgreementQuestion={false}
-                firstAnswer="How the website looks and how easy it is for the users"
-                secondAnswer="Logic behind how the website is built"
+                isAgreementQuestion={
+                  QuizQuestions[currentOn].isAgreementQuestion
+                }
+                firstAnswer={QuizQuestions[currentOn].firstAnswer}
+                secondAnswer={QuizQuestions[currentOn].secondAnswer}
               />
             </div>
           </div>
         </div>
         <div className="button_page">
-          <Buttons label="Next" size="big" isMono={false} />
+          {currentOn !== 0 && currentOn !== QuizQuestions.length - 1 && (
+            <span className="back_button">
+              <Buttons
+                label="Back"
+                color="grey"
+                size="big"
+                isMono={false}
+                onClick={goToPrevious}
+              />
+            </span>
+          )}
+          {currentOn !== QuizQuestions.length - 1 ? (
+            <span className="next_button">
+              <Buttons
+                label="Next"
+                size="big"
+                isMono={false}
+                onClick={handleQuestions}
+              />
+            </span>
+          ) : (
+            <span className="results">
+              <Link to="resultPage">
+                <Buttons label="See My Results" size="big" isMono={false} />
+              </Link>
```

Comment:
This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an `<a>` tag around a `<button>` tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory

Reply:
This is not solved.